mix.exs

```diff
@@ -28,6 +28,7 @@ defmodule UeberauthTwitter.Mixfile do
      {:httpoison, "~> 1.0"},
      {:oauther, "~> 1.1"},
      {:ueberauth, "~> 0.6"},
+     {:hackney, "~> 1.16", override: true},
```

Comment:
I think `hex` does not let you publish a package when you use `override: true` in deps